### ISSUE TITLE
과제 03 재채점

### ISSUE THREAD

**Issue by alxest** [snu-sf-class/pp201602#77]
과제 03의 3번 채점 스크립트를 수정하여 재채점했습니다. 여전히 문제가 있으신 분은 알려주세요.
Status: MISSING FILE
Message:
[Errno 2] No such file or directory: '/tmp/tmpa_zu6gnq/run/src/main/scala/default.test'
위 에러 메세지는 무시하셔도 됩니다.

**Comment by rhs0266** [comment]
3번 채점 결과에 
Status: MISSING FILE
이 떠있는데, 이게 정상이라는 것인가요..?

**Comment by jeuxdeau** [comment]
649번을 수정해서 756번으로 다시 제출했는데, 649번만 MISSING FILE로 채점 되었고 756번은 RUNNING입니다. 확인 부탁드립니다.

**Comment by alxest** [comment]
@rhs0266 
본문에 언급한 
```
 Status: MISSING FILE
Message:
[Errno 2] No such file or directory: '/tmp/tmpa_zu6gnq/run/src/main/scala/default.test'
```
에러가 뜨지 않도록 수정했습니다. 그리고 다시 재채점을 진행 중입니다.
@jeuxdeau 제가 원문에서 말을 애매하게 했는데요, 재채점이 끝난 것이 아니라 재채점을 시작한 것이었습니다. 확인 결과 지금은 756이 채점되어있습니다.
RUNNING은 돌고 있는 중을 의미합니다. 앞으로 RUNNING이 뜬 경우 10분 정도 기다려보시고 그 상태가 계속되면 다시 말씀해주세요. (테스트 케이스당 최대 10초 제한을 두고 50회 정도 테스트를 하니 이론적으로 10분 안에는 끝나야합니다.)

**Comment by jeuxdeau** [comment]
네, 말씀하신 대로 채점이 된 것을 확인했습니다. 감사합니다. 테스트 케이스가 외계어로 표시되는 것은 왜인지 알 수 있을까요? 인코딩 문제인가 했는데 설정을 바꾸어 보아도 해결되지 않아서 여쭤봅니다.

**Comment by rhs0266** [comment]
3번 채점 결과를 보면 글씨가 전부 깨져서 보기 힘든데 혹시 수정 가능한가요?

**Comment by alxest** [comment]
@jeuxdeau 스칼라의 기본 Random.nextString의 결과입니다.

**Comment by alxest** [comment]
@rhs0266 가능은 합니다만 이게 중요한 문제인지 잘 모르겠군요..
